consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -609,6 +617,7 @@ impl<
                 mint_config_tx.prefix.tombstone_block > expired_block_index
             }
             ConsensusValue::MintTx(mint_tx) => mint_tx.prefix.tombstone_block > expired_block_index,
+            ConsensusValue::TimeStamp(_) => true,
```

Comment:
Why retain the timestamp for the next block?

Reply:
no longer present in new work of #3733